.gitignore

```diff
@@ -1,23 +1,20 @@
 tmp
 examples
 node_modules
-static/js/config-jodi.js
-builds/bfe.dev.js.typeahead
+
 jodi.html
-static/profiles/bibframe/Test profile (jodiw).json
+test.html
 
-builds/bfe.dev.bak
-builds/bfe.jodi.bak
-builds/bfe.jodi.js
-static/profiles/bibframe/Test All (jodiw).json
-static/profiles/bibframe/profiles.json
+builds/*
 
-builds/bfe.test.js
+static/js/config-jodi.js
```

Comment:
Do we want to instead to ignore `static/js/config-*.js` in case we use other temporary config files? It's also kind of weird (to me at least) to have specific references to one person's bespoke files.

Reply:
I'm going to merge this PR and then do more fixes; I'll include that in the next commit.